src/styles/index.scss

```diff
@@ -122,10 +124,13 @@ h3 {
 
 table {
   width: 85%;
+  border-collapse: separate;
```

Comment:
Added this 'back' in. When we put in the reset code it did funky things to the formatting of our table. This gets it back to where we like it.